constants.py

```diff
@@ -153,7 +154,7 @@ class ClientType:
         ),
     )
     MOBILE_WEB = ClientInfo(
-        URL("https://m.twitch.tv"),
+        BASE_URL_MOBILE,
```

Comment:
Note: This URL from `ClientInfo` class already derefs this value into a `CLIENT_URL` at the top of `twitch.py`. With a new attribute added, this value could then be used anywhere in the application, instead of being imported as a constant.